Titanium.Web.Proxy/Network/Certificate/BCCertificateMaker.cs

```diff
@@ -145,8 +148,8 @@ public X509Certificate2 MakeCertificate(string sSubjectCn, bool isRoot, X509Cert
             if (!doNotSetFriendlyName)
             {
                 try
-                {
-                    x509Certificate.FriendlyName = subjectName;
+                { 
+                    x509Certificate.FriendlyName = System.Text.RegularExpressions.Regex.Replace(subjectName.ToLower(), @"^CN\s*=\s*", ""); 
```

Comment:
ToLower and the expresion contains upper cased CN? Anyway you should use case insensitive compiled regex.